app/src/main/java/org/oppia/app/topic/TopicFragment.kt

```diff
@@ -11,13 +11,15 @@ import javax.inject.Inject
 /** Fragment that contains tabs for Topic. */
 class TopicFragment : InjectableFragment() {
   @Inject lateinit var topicFragmentPresenter: TopicFragmentPresenter
+  private lateinit var topicId: String
 
   override fun onAttach(context: Context?) {
     super.onAttach(context)
     fragmentComponent.inject(this)
   }
 
   override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-    return topicFragmentPresenter.handleCreateView(inflater, container)
+    topicId = arguments!!.getString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY)
```

Comment:
There is warning on this line. Mainly `topicId` is non-null and the argument can provide a null value. So type mismatch warning.

Reply:
Added a null check and as per our discussion we are avoiding the warning for now.